### ISSUE TITLE
Dawn on ARM?

### ISSUE THREAD

**Issue by philthane** [ghostery/user-agent-desktop#487]
Dawn works fine on my Linux Mint x64 desktop. Any chance someone could build it for ARM64? I have a Pinebook Pro running Manjaro. Willing to test & report if it helps?

**Comment by PoorPocketsMcNewHold** [comment]
I could give this [bugzilla.mozilla.org/show\_bug.cgi?id=1450742](https://bugzilla.mozilla.org/show_bug.cgi?id=1450742)

**Comment by chrmod** [comment]
We would love to offer Linux ARM builds. Would be great if community can help us to do it.

**Comment by torspedia** [comment]
> Dawn works fine on my Linux Mint x64 desktop. Any chance someone could build it for ARM64? I have a Pinebook Pro running Manjaro. Willing to test & report if it helps?

Yup, I'd be interested in this as well, for I have a PBP also running Manjaro Arm (KDE).
